app/components/fetch_month_component.html.erb

```diff
@@ -1,5 +1,5 @@
 <tr>
   <td><%= year %></td>
   <td><%= month %></td>
-  <td><%= status %></td>
+  <td><%= render(StatusComponent.new(status_obj: fetch_month)) %></td>
```

Comment:
Switched to a separate StatusComponent so that could be used for both FetchMonth and RegistrationJob.